.github/workflows/jetbrains-updates.yml

```diff
@@ -6,95 +6,23 @@ on:
 
 jobs:
```

Comment:
@corneliusludmann could you add `workflow_dispatch` that we can trigger it manually as well?